pkg/virt-controller/watch/vm/fuzz/fuzz_test.go

```diff
@@ -0,0 +1,376 @@
+package fuzz
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	stdruntime "runtime"
+	"testing"
```

Comment:
Can you make use of gomega/ginkgo to be consistent with the rest of the codebase?